### DISCUSSION TITLE
copy A to B on a plugin

### DISCUSSION THREAD

**Discussion by emerymastering** [apohl79/audiogridder#1026] [Ideas]
Hello Alex,

I hope you doing good!

I was wondering if implementing a copy of setting from A to B and back is on your todo list?:)

Would be very helpful!

Cheers
Denis

**Comment by emerymastering** [comment]
Oioi, is this a possible feature to implement at all?)

**Comment by apohl79** [comment]
yes, created a feature request ticket a few days back. thx for the suggestion.
